packages/react/src/components/Popover/index.tsx

```diff
@@ -98,6 +98,11 @@ interface PopoverBaseProps {
    */
   align?: PopoverAlignment;
 
+  /**
+   * Will add padding value to arrow in case of bottom-left, bottom-right or top-left, top-right
```

Comment:
I think we should avoid specifying the deprecated alignment options. Keeping it experimental for now seems prudent, too. Maybe revising to something like this:

```suggestion
   * **Experimental**: Specify padding between the caret and the popover edge. 
   * This helps avoid the caret visually separating from Popovers with rounded 
   * corners when aligned bottom-start, bottom-end, top-start, or top-end.
```